examples/gno.land/r/gov/dao/v3/memberstore/memberstore.gno

```diff
@@ -0,0 +1,105 @@
+package memberstore
+
+import (
+	"std"
+
+	"gno.land/p/demo/avl"
+	"gno.land/r/gov/dao/proxy"
+)
+
+var members MembersByTier
+var Tiers TiersByName
+
+const (
+	T1 = "T1"
+	T2 = "T2"
+	T3 = "T3"
+)
+
+func init() {
+	members = NewMembersByTier()
+
+	Tiers = TiersByName{avl.NewTree()}
+	Tiers.Set(T1, Tier{
+		InvitationPoints: 3,
+		Payment:          90000,
+		MinSize: func(membersByTier MembersByTier, tiersByName TiersByName) int {
+			return 70
```

Comment:
Is it planned to implement the "time based requirements"? I mean `at least 70 after 7 years`, `2 new T1 members per quarter`, etc.

Reply:
Ok, I don't see how to do it either. Maybe in this case it would require to have oracles to check the real time? But even with that, TBH I don't understand what should happen concretely if this condition is not met. Nor how to check this condition periodically, etc. 
Maybe this point should be removed from the specs then?